pkg/controller/account/ec2.go

```diff
@@ -85,8 +85,8 @@ func (r *ReconcileAccount) InitializeRegion(reqLogger logr.Logger, account *awsv
 	return nil
 }
 
-// BuildandDestroyEC2Instances runs and ec2 instance and terminates it
-func (r *ReconcileAccount) BuildandDestroyEC2Instances(reqLogger logr.Logger, account *awsv1alpha1.Account, awsClient awsclient.Client, ami string) error {
+// BuildAndDestroyEC2Instances runs and ec2 instance and terminates it
```

Comment:
This change isn't related to the metrics change. Please remove it

Reply:
Thanks for the review. I already put this change in a separate commit. Is it okay?